# HG changeset patch
# Parent 807be0c0123627aceb90dfa3d9a39938387261d4
Make 2html.vim auto link URL which starts 'ftp'

diff -r 807be0c01236 runtime/syntax/2html.vim
--- a/runtime/syntax/2html.vim	Sat Apr 13 22:36:57 2013 +0900
+++ b/runtime/syntax/2html.vim	Sat Apr 13 22:43:04 2013 +0900
@@ -1633,7 +1633,7 @@
 " Add hyperlinks
 " TODO: add option to not do this? Maybe just make the color the same as the
 " text highlight group normally is?
-%s+\(https\=://\S\{-}\)\(\([.,;:}]\=\(\s\|$\)\)\|[\\"'<>]\|&gt;\|&lt;\|&quot;\)+<a href="\1">\1</a>\2+ge
+%s+\(\%(https\=\|ftp\)://\S\{-}\)\(\([.,;:}]\=\(\s\|$\)\)\|[\\"'<>]\|&gt;\|&lt;\|&quot;\)+<a href="\1">\1</a>\2+ge
 
 " The DTD
 if s:settings.use_xhtml
